application/views/survey/questions/answer/arrays/multiflexi/rows/cells/answer_td_checkboxes.twig

```diff
@@ -25,6 +25,6 @@
         'aria-labelledby' : "answertext" ~ basename ~ "_" ~ dataCode,
     }))
     }}
-    <label for="cbox{{ myfname2 }}" class="ls-label-xs-visibility" aria-hidden="true">{{ dataTitle }}</label>
+    <label for="cbox{{ myfname2 }}" class="ls-label-xs-visibility" aria-hidden="true">{{ processString(dataTitle) }}</label>
```

Comment:
Should this be done here or maybe in another layer? (like PHP)
Just asking, as this may also happen for other question types ...